### PR TITLE
Itsadssd 29284 component documentation

### PR DESCRIPTION
**Pull request by Bcrunk**

### CODE REVIEW

**Review by jen-lui — CHANGES_REQUESTED** [review]

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
+
+# Blockquote
+
+<StatusBadge.Review /><StatusBadge.ReviewDoc />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:10]: I hate using BR tags to add space ... I might move this to a new story for us to change the settings in storybook.

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+For quoting a passage of text or speech from another source.
+
+## Default example
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:14]: Updated,

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+<Canvas withSource="none">
+  <Story id="components-blockquote" />
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:17]: Updated. 

There is an issue with the height of the canvases, it works automatically in some instances and not in others. I will review and see if i fix here or create a new story for that system wide

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        <p>Words can be like X-rays, if you use them properly — they’ll go through anything. You read and you’re pierced.</p>
+        <p>Words can be like X-rays, if you use them properly — they’ll go through anything. You read and you’re pierced.</p>
+    </div>
+    <div class="blockquote__footer">
+        <p>Aldous Huxley, <cite><a href="#" class="blockquote__link">Brave New World</a></cite></p>
+    </div>
+</blockquote>
+```
+
+## Variant examples
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:32]: The initial task was just to correct the structure, I had to stop myself from going further and reworking all the examples, that is covered in the uplift tickets. But I agree and i think some of the examples and variants should be dropped, the Quote variant does not add value.

File: packages/storybook-html/src/components/blockquote/blockquote.stories.js [hunk 5]
```diff
@@ -3,16 +3,71 @@ import './blockquote.scss';
 
 // import HTML template strings
 import blockquoteHTML from './blockquote.html';
+import blockquoteDarkHTML from './blockquote_dark.html';
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.stories.js:6]: Do you have an example.. I believe the the underscore is a requirement of storybook not optional in the naming structure.

File: packages/storybook-html/src/components/blockquote/blockquote.stories.js [hunk 6]
```diff
@@ -3,16 +3,71 @@ import './blockquote.scss';
...
+export const blockquoteDark = () => {
+  return blockquoteDarkHTML;
+};
+blockquoteDark.storyName = 'Dark';
+blockquoteDark.parameters = {
+  docs: {
+     inlineStories: false
+  },
+  backgrounds: {
+    default: 'Dark'
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.stories.js:43]: Agree naming is off for this aspect, I think it needs a significant correction.  I will add notes to the uplift story

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+```html
+<blockquote class="blockquote blockquote--dark blockquote--boxed blockquote--quotemark">
+    <div class="blockquote__body">
+        <p>Words can be like X-rays, if you use them properly — they’ll go through anything. You read and you’re pierced.</p>
+    </div>
+    <div class="blockquote__footer">
+        <p>Aldous Huxley, <cite><a href="#" class="blockquote__link">Brave New World</a></cite></p>
+    </div>
+</blockquote>
+```
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:117]: I have correct the structure of the existing doc, I did not add in the missing sections but added them to he uplift.

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:32]: added to uplift

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:17]: Added a story to review canvas heights in storybook.

File: packages/storybook-html/src/components/blockquote/blockquote.stories.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.stories.js:43]: added to uplift

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx (hunk 7 above)
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:117]: added to uplift

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+## Default example
+
+<Canvas withSource="none">
+  <Story id="components-blockquote" />
+</Canvas>
+
+```html
+<blockquote class="blockquote blockquote--light">
+    <div class="blockquote__body">
+        <p>Words can be like X-rays, if you use them properly — they’ll go through anything. You read and you’re pierced.</p>
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:23]: added to Uplift sotry

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:10]: I have added a story into backlog to evaluate system wide.

File: packages/storybook-html/src/components/blockquote/blockquote.stories.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (jen-lui) [reply packages/storybook-html/src/components/blockquote/blockquote.stories.js:6]: Table, button, accordion - the HTML files use hyphen

[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.stories.js]: Updated

File: packages/storybook-html/src/components/blockquote/blockquote.docs.mdx [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+    </div>
+    <div class="blockquote__footer">
+        <p>Aldous Huxley, <cite><a href="#" class="blockquote__link">Brave New World</a></cite></p>
+    </div>
+</blockquote>
+```
+
+## Variant examples
+
```
[the remark this answers is not in the document]
Comment (Bcrunk) [reply packages/storybook-html/src/components/blockquote/blockquote.docs.mdx:34]: added to uplift story

**Review by jen-lui — APPROVED** [review]

**Pull request merged**